spanner/api/SpannerTest/Tests.cs

```diff
@@ -133,13 +128,83 @@ public SpannerTests(SpannerFixture fixture)
                 if (!_fixture.s_initializedDatabase)
                 {
                     _fixture.s_initializedDatabase = true;

```

Comment:
Move this to after all the initializations have happened, just in case some of hem fail. On rewrite we will move all of this to the fixture, and there won't be a need to ask whether initialization has happened or not.

Reply:
This will remove in upcoming rewrite.